docs/editing.md

```diff
@@ -151,7 +151,7 @@ Sometimes you'll need to add a new reviewer once the main review (i.e. post pre-
 
 ## Post-review
 
-When a submission is ready to be accepted, we ask that the authors issue a new tagged release of the software (if changed), and archive it (on [Zenodo](https://zenodo.org/), [fig**share**](https://figshare.com/), or other). The authors then post the version number and archive DOI in the `REVIEW` issue. The handling editor executes the pre-publication steps, and pings the Track Editor-in-Chief for final processing.
+When a submission is ready to be accepted, we ask that the authors issue a new tagged release of the software (if changed), and archive it on a service that ideally supports key features such as software licenses, author ORCIDs, and version tagging aligned with the actual content (rather than merely reflecting the number of archive deposits). One suitable example is [Zenodo](https://zenodo.org/). The authors then post the version number and archive DOI in the `REVIEW` issue. The handling editor executes the pre-publication steps, and pings the Track Editor-in-Chief for final processing.
```

Comment:
I don't think we can say "ideally". We need to state firm requirements.  And I don't really understand a lot of the text here. 

Also, I don't think "version tagging aligned with the actual content (rather than merely reflecting the number of archive deposits)" is really a requirement. If this is just a preference, we should drop it from the text.